scripts/e2e-email/e2e-email.js

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const assert = require('assert')
+const crypto = require('crypto')
+const url = require('url')
+
+const P = require('../../promise')
+const preq = require('./p-request')
+const localeQuirks = require('./localeQuirks')
+
+var program;
+try {
+  program = require('commander');
+} catch(e) {
+  console.log("This program requires you to do `npm install commander` first.");
+  process.exit(1);
+}
+
+program
+  .option('-a, --auth-server [url]',
+          'URL of FxA Auth Server',
+          'https://api-accounts.stage.mozaws.net')
+  .option('-r, --restmail-domain [fqdn]',
+          'URL of the restmail server',
+          'restmail.net')
+  .option('-l, --locales [path]',
+          'Path to list of locales to test',
+          '../../../fxa-content-server/server/config/production-locales.json')
+  .option('-m, --max-locales [int]', // useful for smoketest and debugging
+          'Only send find email for first M locales [Infinity]',
+          parseInt, Infinity)
+  .option('-L, --lock-account',
+          'Should test include account locking (option not implemented)')
```

Comment:
Yeah, that. Skipping that part of the flow not implemented. I kinda meant that as a hint should someone try to use this with prod and have it utterly fail. But, yeah, I'm gonna remove that option, and just leave a comment warning about the limitation.